spec/models/minion_spec.rb

```diff
@@ -4,29 +4,61 @@
 describe Minion do
   it { is_expected.to validate_uniqueness_of(:hostname) }
 
+  # rubocop:disable RSpec/ExampleLength
```

Comment:
which method? afaics it means that there are too many examples, but what is the problem with that?